aviary/docs/getting_started/installation.md

```diff
@@ -1,317 +1,268 @@
 # Installation
 
-## Quick start installation
+Aviary can be installed on any OS, including Windows, Linux, and MacOS. We recommend using a conda environment, as it makes it easier to install Aviary with it's full feature set, which will be explained below.
 
-```{note}
-If you do not already have Python installed, we recommend installing [condaforge](https://github.com/conda-forge/miniforge?tab=readme-ov-file#download).
-The minimum supported version of Python is 3.9; we recommend using the latest release of Python.
-```
+We assume that you have a working Python or conda environment set up for all of our guides. Our recommended package manager is [miniforge](https://github.com/conda-forge/miniforge). Miniforge is an open-source alternative to miniconda, and functions identically.
 
-The simplest way to install Aviary to use pip in a terminal:
+The Detailed and Developer's installation guide also assume you have git installed. The Developer's Guide also assumes that if you are using your own fork of Aviary, that you have your machine correctly configured with the proper permissions to pull from and push to your fork.
 
-```python
-pip install aviary
+
+```{note}
+The minimum supported version of Python is 3.9; we recommend using the latest release of Python.
 ```
 
-This will install the latest release of Aviary and all of its dependencies.
+## Which Installation Guide Should I Use?
+The different installation guides are intended to get users of various levels of experience and analysis need started. Here is a brief explanation of what can differ between Aviary installs.
 
-That's it!
-If you've done that successfully, you can now use Aviary in your Python environment.
+### Optimizers
+Aviary uses the pyOptSparse package to get access to a variety of different optimizers. Many of the more powerful and robust optimizers do not come with pyOptSparse by default, and may require additional steps to install. You can always "upgrade" your pyOptSparse later with more optimizers. Using a conda environment simplifies that installation process.
```

Comment:
Many of the more powerful and robust optimizers do not come with pyOptSparse by default, and may require additional steps to install. 

I think we should add - (Note: Some of the most powerful optimizers like SNOPT require a Linux installation.)